Jenkinsfile

```diff
@@ -20,7 +20,7 @@ pipeline {
                 stage('Test') {
                     steps {
                         // biopython gets imported as 'Bio' and scikit-image gets imported as 'skimage'
-                        sh 'podman run -it --rm localhost/$IMAGE_NAME python -c "import numpy; import scipy; import matplotlib; import pandas; import Bio; import statsmodels; import skimage; import seaborn; import datetime; import os; import shutil"'
+                        sh 'podman run -it --rm localhost/$IMAGE_NAME python -c "import numpy; import scipy; import matplotlib; import pandas; import Bio; import statsmodels; import skimage; import seaborn; import datetime; import os; import shutil; import torch; import torchvision"'
```

Comment:
Can we also `import torchaudio` just to make sure our bases are covered for matching python, torch, and torchaudio compatibility?  Otherwise, LGTM.